.github/workflows/php-test-plugins.yml

```diff
@@ -42,10 +42,10 @@ jobs:
       fail-fast: false
       matrix:
         php: ['8.2', '8.1', '8.0', '7.4', '7.3', '7.2']
-        wp: [ 'latest' ]
+        wp: [ '6.6-RC2' ] # TODO: Change this back to latest.
```

Comment:
We'll switch this back to `latest` once the 6.6 release is finalized. In the meantime, `trunk` is basically the same, so we're not hurting from not testing a WP version.